types/maxmsp/maxmsp-tests.ts

```diff
@@ -46,11 +46,29 @@ if (!folder.end) {
 folder.close();
 
 // maxmsp-tests.ts
+// 1. working with Global
+const global = new Global("myGlobal");
+global.prop = "hello";
+// @ts-expect-error
+messnamed("myobj", global.prop);
+if (typeof global.prop !== "string") {
+	throw new Error("prop is not a string");
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/maxmsp/maxmsp-tests.ts;
  Line: 55;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.